### DISCUSSION TITLE
[Freya] Latests versions issues (3.47 & 3.48)

### DISCUSSION THREAD

**Discussion by Y0niix** [primefaces/community#1114] [PrimeVue Templates]
Dear all,
We bought the Freya theme a few months ago, we are still on version 4.0.0 for our project (API options).
Everything was working fine until we updated the Primevue package to version 3.47. 

I saw that there was a breaking change for issue 5071 (Switch no longer works, Multiselect style, radio buttons, etc.).

Do you have a solution? 
Why don't the components work anymore? How can I get them to work again with version 3.47.2? and even version 3.48.0?

Why and how can paid themes not be maintained at the same time as PrimeVue?

Thank you for your answers.

**Comment by lucianovalenca** [comment]
same here... In Apollo I had to keep primevue on 3.38.1, otherwise a lot of things breaks

**Comment by petrbouda** [comment]
I have issues with SCSS compilation: https://github.com/orgs/primefaces/discussions/1150
